content/tbb/tbb-29/contents.lr

```diff
@@ -7,7 +7,7 @@ seo_slug: change-ip-address
 description:
 Tor Browser has two ways to change your relay circuit — "New Identity" and "New Tor Circuit for this Site".
 
-Both options are located in the Menu, but you can also access the New Circuit option inside the site information menu, in the URL bar.
+Both options are located in the hamburger menu, but you can also access the New Circuit option by clicking the small sparky broom icon at the top-right of the screen.
```

Comment:
New Circuit and New Identity are two different things.

Replace this paragraph to something like this:

Both options are located in the hamburger menu. You can also access the **New Circuit** option inside the site information menu, in the URL bar and the **New Circuit** option by clicking the small sparky broom icon at the top-right of the screen.

Reply:
> New Circuit and New Identity are two different things.
> 
> Replace this paragraph to something like this:
> 
> Both options are located in the hamburger menu. You can also access the **New Circuit** option inside the site information menu, in the URL bar and the **New Circuit** option by clicking the small sparky broom icon at the top-right of the screen.

@gusgustavo : i think this is not true. i can only access New Identity from the sparky broom.

Reply:
New Identity and New Circuit are both available in hamburger menu. Fixed there.